pcsx2-qt/GameList/GameListModel.cpp

```diff
@@ -457,12 +457,12 @@ void GameListModel::loadCommonImages()
 	m_type_exe_pixmap = QIcon(QStringLiteral(":/icons/applications-system-24.png")).pixmap(QSize(24, 24));
 	m_type_playlist_pixmap = QIcon(QStringLiteral(":/icons/address-book-new-22.png")).pixmap(QSize(22, 22));
 
-for (u32 i = 0; i < static_cast<u32>(GameList::Region::Count); i++) 
-	{
-		m_region_pixmaps[i] = QIcon(
-								QStringLiteral(":/icons/flags/%1.png").arg(GameList::RegionToString(static_cast<GameList::Region>(i))))
-								.pixmap(QSize(42, 30));
-	}
+	for (u32 i = 0; i < static_cast<u32>(GameList::Region::Count); i++) 
+		{
```

Comment:
brackets should be inline with the for so like

	for(blah)
	{
		//stuff happening
	}

Reply:
🐧 Guess I moved too much, changed that

Reply:
perfect <3